dressup/src/components/Cards/ProdListingCard.js

```diff
@@ -3,19 +3,20 @@ import "./ProdListingCard.css";
 import { FaStar } from "react-icons/fa";
 import { FaHeart } from "react-icons/fa";
 import { FaShoppingCart } from "react-icons/fa";
-
+import { useWishCart } from "../../pages/CartPage/WishCart-context";
+const message = (msg) => {
+  console.log(msg);
+};
 function ProdListingCard(props) {
+  const { stateOne, dispatchOne } = useWishCart();
   return (
```

Comment:
it will be good if you destructure the props.

Reply:
Okay will work on this